README.md

```diff
@@ -0,0 +1,278 @@
+
+# Kafka topic to push notification data
+NOTIFICATION_KAFKA_TOPIC = notificationtopic
+
+# Kafka topic name to consume from mentoring topic
+KAFKA_MENTORING_TOPIC ="mentoringtopic"
+
+# Kafka topic to push recording data
+KAFKA_RECORDING_TOPIC ="recordingtopic"
+
+# Any one of three features available for cloud storage
+CLOUD_STORAGE = 'GCP/AWS/AZURE'
+
+# Gcp json config file path
+GCP_PATH = 'gcp.json'
+
+# Gcp bucket name which stores files
+DEFAULT_GCP_BUCKET_NAME = 'gcp-bucket-storage-name'
+
+# Gcp project id
+GCP_PROJECT_ID = 'project-id'
+
+# Aws access key id
+AWS_ACCESS_KEY_ID = 'aws-access-key-id'
+
+# Aws secret access key
+AWS_SECRET_ACCESS_KEY = 'aws-secret-access-key'
+
+# Aws region where bucket will be located
+AWS_BUCKET_REGION = 'ap-south-1'
+
+# Aws end point
+AWS_BUCKET_ENDPOINT = 's3.ap-south-1.amazonaws.com'
+
+# Aws bucket name which stores files
+DEFAULT_AWS_BUCKET_NAME = 'aws-bucket-storage-name'
+
+# Azure storage account name
+AZURE_ACCOUNT_NAME = 'account-name'
+
+# Azure storage account key
+AZURE_ACCOUNT_KEY = 'azure-account-key'
+
+# Azure storage container which stores files
+DEFAULT_AZURE_CONTAINER_NAME = 'azure-container-storage-name'
+
+# user serice host
+USER_SERIVCE_HOST = 'http://localhost:3001'
+
+# user serice base url
+USER_SERIVCE_BASE_URL = '/user/'
+
+# Big blue button url
+BIG_BLUE_BUTTON_URL = https://dev.mentoring.shikshalokam.org
+
+# Big blue button base url
+BIB_BLUE_BUTTON_BASE_URL = /bigbluebutton/
+
+# Meeting end callback events end point
+MEETING_END_CALLBACK_EVENTS = https%3A%2F%2Fdev.elevate-apis.shikshalokam.org%2Fmentoring%2Fv1%2Fsessions%2Fcompleted
+
+# Big blue button secret key
+BIG_BLUE_BUTTON_SECRET_KEY = n
+
+# Big blue button recording ready callback url
+RECORDING_READY_CALLBACK_URL = http%3A%2F%2Flocalhost%3A3000%2F%3FmeetingID%3Dmeet123
+
+# Enable logging of network request
+ENABLE_LOG = true
+
+# Api doc url
+API_DOC_URL = '/api-doc'
+
+```
+
+### 4. Install Npm
+
+    npm i
+    To install the dependencies in your local machine.
+
+### 5. To Run the server
+
+    npm start
+
+## Setup Mentoring with Docker
+
+Recommend to,
+
+Install any IDE in your system(eg: VScode etc..)
+
+Install Docker Engine from: https://docs.docker.com/engine/install/
+
+Install Docker Compose from: https://docs.docker.com/compose/install/
+
+### 1. Pull Required Images
+
+Go to https://github.com/ELEVATE-Project/mentoring From the code tab copy the link. Using that link clone the repository into your local machine.
+
+Let's make it easier for you:
+
+    1. Create a new folder where you want to clone the repository.
+    2. Goto that directory through the terminal and execute the commands.
+
+git clone https://github.com/ELEVATE-Project/mentoring.git
+
+### 2. Add the .env file to the project directory
+
+    create a file named .env in the root directory of the project and copy the below code into that file.
+    Add the following environment configs
+
+### 3. Required Environment variables:
```

Comment:
@ankitpws .env variables are the same keep it only once 
add note to refer the .env variables